config/config.go

```diff
@@ -155,15 +155,15 @@ func validateAllUnique(c configpb.Configuration) error {
 	return mErr
 }
 
-func validateReferencesExist(c configpb.Configuration) error {
+func validateReferencesExist(c *configpb.Configuration) error {
```

Comment:
If these function arguments are going to change to pointers, they now need null checks.

```
if c == null {
  return fmt.Errorf("descriptive message")
}
```

Similarly for other functions.

Reply:
Fiiiiiiixed I think?

Reply:
In this PR: Michelle learned the importance of using the dang proto methods and not just accessing things willy-nilly.